src/index.html

```diff
@@ -5,8 +5,307 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+    rel="preconnect"
+    href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="moyo logo"
+        >
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartfones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgeds" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="cards">
+        <div class="cards__card card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+            class="card__image"
+          >
+          <h1 class="card__prod">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+          <h2 class="card__code">
+            Product code: 195434
+          </h2>
+          <div class="card__review">
+            <div class="card__review stars">
+              <div class="stars__star star-1"></div>
+              <div class="stars__star star-2"></div>
+              <div class="stars__star star-3"></div>
+              <div class="stars__star star-4"></div>
+              <div class="stars__star star-5"></div>
+            </div>
+            <p class="card__review-rev">Reviews: 5</p>
+          </div>
+          <div class="card__price-container">
```

Comment:
```suggestion
          <div class="card__review">
            <div class="card__review stars">
              <div class="stars__star star-1"></div>
              <div class="stars__star star-2"></div>
              <div class="stars__star star-3"></div>
              <div class="stars__star star-4"></div>
              <div class="stars__star star-5"></div>
            </div>
            
            <p class="card__review-rev">Reviews: 5</p>
          </div>
          
          <div class="card__price-container">
```
add spaces between multiline blocks

Reply:
2'nd point in checklist